.github/workflows/release.yml

```diff
@@ -26,12 +26,13 @@ jobs:
 
     - name: Release
       shell: bash
-      env: 
+      env:
         MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
         GH_TOKEN: ${{ secrets.GH_TOKEN }}
         GH_USER: ${{ secrets.GH_USER }}
         APPLE_ISSUER_ID: ${{ secrets.APPLE_ISSUER_ID }}
         APPLE_KEY_ID: ${{ secrets.APPLE_KEY_ID }}
         APPLE_KEY_CONTENT: ${{ secrets.APPLE_KEY_CONTENT }}
+        WALLETAPP_SENTRY_DNS: ${{ secrets.WALLETAPP_SENTRY_DNS }}
```

Comment:
I believe it's actually `DSN` not `DNS` 😅

Reply:
😅 my bad